pkg/odo/commonflags/context/context.go

```diff
@@ -0,0 +1,49 @@
+package context
+
+import (
+	"context"
+
+	"github.com/redhat-developer/odo/pkg/odo/commonflags"
+)
+
+const (
+	OutputFlag = "outputFlag"
+)
+
+type (
+	outputKeyType struct{}
+	runOnKeyType  struct{}
+)
+
+var (
+	outputKey outputKeyType
+	runOnKey  runOnKeyType
+)
+
+// WithOutput sets the value for the output flag (-o) in ctx
+func WithOutput(ctx context.Context, val bool) context.Context {
```

Comment:
Same question: to make the intent clearer, would it make sense to rename this into something like `WithJsonOutput`?